internal/v2/api/handlers/stats.go

```diff
@@ -44,3 +45,25 @@ func (h *V2Handler) GetOverallStats(request *http.Request) (*handler.Result, *ty
 	}
 	return handler.NewResult(stats), nil
 }
+
+// GetPrices @Summary Get latest prices for all available symbols
+// @Description Get latest prices for all available symbols
+// @Produce json
+// @Tags v2
+// @Success 200 {object} handler.PublicResponse[map[string]float64] ""
```

Comment:
Not sure that it's correct way to specify response, but it is what it is (just map of string to float64)

Reply:
Let's leave it as is for now. For now it's flexibility vs hardcode let's see how it will play out